core/src/main/java/org/geysermc/geyser/inventory/recipe/TrimRecipe.java

```diff
@@ -46,18 +45,18 @@ public final class TrimRecipe {
     public static final ItemDescriptorWithCount ADDITION = tagDescriptor("minecraft:trim_materials");
     public static final ItemDescriptorWithCount TEMPLATE = tagDescriptor("minecraft:trim_templates");
 
-    public static TrimMaterial readTrimMaterial(GeyserSession session, RegistryEntry entry) {
-        String key = entry.getId().asMinimalString();
+    public static TrimMaterial readTrimMaterial(RegistryContext context) {
```

Comment:
nitpick, but, i would either keep these param names as `entry`, or add javadoc to RegistryContext that it is specific to the entry it contains (and is not a context for the whole registry)

Reply:
I added a simple Javadoc to the RegistryContext record